### ISSUE TITLE
Support for browser only (Svelte + Vite)?

### ISSUE THREAD

**Issue by nine-2-five** [microsoft/kiota-typescript#1381]
I have successfully tested a Kiota Typescript client in a standalone Node project ran with npx tsx index.ts.
However my requirement is to call this client from a Svelte (Vite bundled) app, and that is browser only.
Is there any chance this could happen, with current standings?

The plan was to use it from a Svelte SPA running inside MAUI HybridWebView.

**Comment by baywet** [comment]
Hi @nine-2-five 
Thank you for using kiota and for reaching out.

While we have no tested this specific framework, the dependencies and the generated code are supposed to work both in a node or browser context.
Let us know if you have any additional comments or questions.

**Comment by nine-2-five** [comment]
I got it to compile, but this pops out in browser:

```
Uncaught SyntaxError: The requested module '/node_modules/.vite/deps/@microsoft_kiota-abstractions.js?v=80e0625c' does not provide an export named 'UntypedNode' (at index.ts:5:135)
```

**Comment by baywet** [comment]
Thank you for the additional information.

Can you run `npm list @microsoft/kiota-abstractions` and share what results you get please?

**Comment by nine-2-five** [comment]
Sure, this the result:

├── @microsoft/kiota-abstractions@1.0.0-preview.66
├─┬ @microsoft/kiota-http-fetchlibrary@1.0.0-preview.65
│ └── @microsoft/kiota-abstractions@1.0.0-preview.66 deduped
├─┬ @microsoft/kiota-serialization-form@1.0.0-preview.54
│ └── @microsoft/kiota-abstractions@1.0.0-preview.66 deduped
├─┬ @microsoft/kiota-serialization-json@1.0.0-preview.66
│ └── @microsoft/kiota-abstractions@1.0.0-preview.66 deduped
├─┬ @microsoft/kiota-serialization-multipart@1.0.0-preview.44
│ └── @microsoft/kiota-abstractions@1.0.0-preview.66 deduped
└─┬ @microsoft/kiota-serialization-text@1.0.0-preview.63
  └── @microsoft/kiota-abstractions@1.0.0-preview.66 deduped

**Comment by baywet** [comment]
Thank you for the additional information.
I wanted to confirm you were running on the latest version of the package.

Since UntypedNode is an interface, my guess is that it gets "erased at transpilation" but somehow something still refers to it. In the generated client, can you search for any reference to UntypedNode please and share how they are used here please?

**Comment by nine-2-five** [comment]
It is referenced on multiple lines in client/models.ts (I'm using Google Place object)

export interface GOpeningHours extends Parsable {
    /**
     * The open_now property
     */
    openNow?: boolean | null;
    /**
     * The periods property
     */
    periods?: Period[] | null;
    /**
     * The weekday_text property
     */
    weekdayText?: UntypedNode | null;
}

**Comment by baywet** [comment]
Thank you for the additional information.

Can you also share the import for the untyped node please?

**Comment by nine-2-five** [comment]
`import { createUntypedNodeFromDiscriminatorValue, type AdditionalDataHolder, type Parsable, type ParseNode, type SerializationWriter, UntypedNode } from '@microsoft/kiota-abstractions';`

**Comment by baywet** [comment]
Thank you for the additional information.
That's the issue, the import for UntypedNode should have a type marker as well.
Authored https://github.com/microsoft/kiota/pull/5477 to address the issue in the generator.

**Comment by nine-2-five** [comment]
Do you know when this will be available in NPM?

**Comment by baywet** [comment]
It's not a change of the NPM package but of the generator. 
Once the change is available with release 1.19 planned to go out on October the 4th, you'll need to refresh your client.

**Comment by nine-2-five** [comment]
Ah, sorry, it thought I installed it globally by npm, but it was installed by dotnet cmd.
